src/esm_runscripts/compute.py

```diff
@@ -462,28 +463,16 @@ def update_runscript(fromdir, scriptsdir, tfile, gconfig, file_type):
                 )
                 correct_input = False
                 while not correct_input:
-                    update_choice = input(
+                    update_choice = questionary.confirm(
                         f"Do you want that {scriptsdir + '/' + tfile} is "
-                        + "updated with the above changes? (y/n): "
-                    )
-                    if update_choice == "y":
+                        + "updated with the above changes?"
+                    ).ask()
+                    if update_choice:
                         correct_input = True
                         oldscript = fromdir + "/" + tfile
                         print(oldscript)
                         shutil.copy2(oldscript, scriptsdir)
                         print(f"{scriptsdir + '/' + tfile} updated!")
-                    elif update_choice == "n":
-                        correct_input = True
-                        esm_parser.user_error(
-                            f"Original {file_type} different from target",
-                            differences
-                            + "\n"
-                            + "You can choose to -U flag in the esm_runscripts call "
-                            + "to update the runscript without asking (WARNING: This "
-                            + f"will overwrite your {file_type} in the experiment folder!)\n\n",
-                        )
-                    else:
-                        print(f"'{update_choice}' is not a valid answer.")
```

Comment:
My brain might be burnt by now... but what happens if the user selects `n`?

Reply:
it's already merged....but if you push `n`, you just get asked the question again until you push C-c. That might be better to just exit instead. I'll fix that in a bit if we want that behavior

Reply:
I think it is better to let the user decide what to do, if you keep asking the same question again and again after they set `n` many users will eventually go for `y`.

Reply:
by the way, using `questionary` we can get rid of the while loop as it automatically detects wrong answers and asks again right?